packages/manager/modules/pci/src/projects/project/storages/storages.module.js

```diff
@@ -2,7 +2,7 @@ import angular from 'angular';
 import '@ovh-ux/ui-kit';
 import '@ovh-ux/manager-filters';
 
-import blocks from './blocks';
+import pciBlockStorage from '@ovh-ux/manager-pci-block-storage';
```

Comment:
not needed as a micro app is independent of other apps

Reply:
see https://github.com/ovh/manager/pull/10563#discussion_r1472797974